<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfForm</class>
 <widget class="QWidget" name="ConfForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>446</width>
     <height>581</height>
    </rect>
   </property>
   <widget class="QWidget" name="wgManual" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>446</width>
      <height>280</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="leAddr">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>15</y>
       <width>250</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QComboBox" name="cbMask">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>60</y>
       <width>250</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="leGateway">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>105</y>
       <width>250</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="leDns">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>150</y>
       <width>250</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="lbTxt2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans CJK SC</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbTxt3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>65</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans CJK SC</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbTxt4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans CJK SC</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbTxt5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>155</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans CJK SC</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="leDns2">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>195</y>
       <width>250</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="lbTxt6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans CJK SC</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="wdHead" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>446</width>
      <height>200</height>
     </rect>
    </property>
    <widget class="QComboBox" name="cbType">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>182</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="lbTxt1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>135</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans CJK SC</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans CJK SC</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="leName">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>75</y>
       <width>290</width>
       <height>48</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QWidget" name="upTitle" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>446</width>
       <height>48</height>
      </rect>
     </property>
     <widget class="QPushButton" name="btnClose">
      <property name="geometry">
       <rect>
        <x>408</x>
        <y>8</y>
        <width>24</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lbLeftupTitle">
      <property name="geometry">
       <rect>
        <x>24</x>
        <y>10</y>
        <width>240</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="isDHCP">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>130</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="notDHCP">
     <property name="geometry">
      <rect>
       <x>356</x>
       <y>130</y>
       <width>50</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="wdBottom" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>446</width>
      <height>96</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btnOk">
     <property name="geometry">
      <rect>
       <x>312</x>
       <y>25</y>
       <width>112</width>
       <height>48</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCancel">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>25</y>
       <width>112</width>
       <height>48</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCreate">
     <property name="geometry">
      <rect>
       <x>312</x>
       <y>25</y>
       <width>112</width>
       <height>48</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="lineUp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>426</width>
      <height>1</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="lineDown">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>426</width>
      <height>1</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
